<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab6CamSecurity</class>
 <widget class="QWidget" name="Tab6CamSecurity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,9">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,3,2">
     <item>
      <widget class="QPushButton" name="pPBCamView">
       <property name="text">
        <string>영상보기</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pPBsnapShot">
       <property name="text">
        <string>스냅샷</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="pCBsecurityMode">
       <property name="text">
        <string>Security Mode</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="pTLcamView">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
